### PR TITLE
29370 unknown service

### LINKED ISSUE #29370: Correct spelling of "unknown_service" in open telemetry
The spec of course requires the correct spelling but currently we have "unkown_service"

### PR DESCRIPTION
**Pull request by benjamin-confino**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

fixes #29370

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
No pipelines from libby

File: dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops [hunk 1]
```diff
@@ -55,6 +55,15 @@ CWMOT5005.mptelemetry.unknown.log.source=CWMOT5005W: The MicroProfile Telemetry
...
+
+CWMOT5007.tel.enabled.conflict=CWMOT5007W: Conflicting configuration for the otel.sdk.disabled configuration property detected for the {0} application. The final value is otel.sdk.disabled=false because the property enabling telemetry for the application overrides the property disabling telemetry for the runtime.
+CWMOT5007.tel.enabled.conflict.explanation: MicroProfile Telemetry can be enabled for the whole server by using system properties or environment variables. If it isn't enabled for the whole server, it can be enabled for an application by setting MicroProfile Config properties; for example, by setting variables in the server.xml file. This warning is given if Telemetry is enabled for the whole server, but the MicroProfile Config properties disable Telemetry for an application. Because Telemetry is enabled for the whole server, the configuration for that application is ignored.
+CWMOT5007.tel.enabled.conflict.useraction: Specify the settings to enable or disable OpenTelemetry instances by using either environment variables and system properties or other MP Config config sources but not both.
```
Comment (helyarp) [inline dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops]: Change "MP Config config sources" to "MP Config sources" (remove "config")

File: dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops [hunk 2]
```diff
@@ -55,6 +55,15 @@ CWMOT5005.mptelemetry.unknown.log.source=CWMOT5005W: The MicroProfile Telemetry
...
 CWMOT5005.mptelemetry.unknown.log.source.useraction=Check the configuration information and specify one of the supported sources for MicroProfile Telemetry.
 
+CWMOT5006.tel.enabled.conflict=CWMOT5006W: Conflicting configuration for the otel.sdk.disabled configuration property detected for the {0} application. The final value is otel.sdk.disabled=false. Telemetry cannot be disabled for an application when it is enabled for the runtime.
+CWMOT5006.tel.enabled.conflict.explanation: MicroProfile Telemetry can be enabled for the whole server by using system properties or environment variables. If it isn't enabled for the whole server, it can be enabled for each application by setting MicroProfile Config properties; for example, by setting variables in the server.xml file. This warning is given if there's an explicit configuration that disables Telemetry for the whole server, which is overridden by an MP Config property that enables Telemetry for an application.
+CWMOT5006.tel.enabled.conflict.useraction: Specify the settings to enable or disable OpenTelemetry instances by using either environment variables and system properties or other MP Config config sources but not both.
```
Comment (helyarp) [inline dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops]: Change "MP Config config sources" to "MP Config sources" (remove "config")

**Review by helyarp — COMMENTED** [review]
Small changes suggested

**Comment by benjamin-confino** [comment]
#build

**Review by helyarp — COMMENTED** [review]
Looks good!

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
trying to get a pipeline

**Comment by benjamin-confino** [comment]
had to rebase on integration due to this needing to update both my other PRs and them being forked from different levels of integration

**Comment by benjamin-confino** [comment]
#build

**Review by Yas-Franco — APPROVED** [review]

**Pull request merged**